src/genophenocorr/view/_protein_visualizable.py

```diff
@@ -1,4 +1,5 @@
 import typing
+import regex as re
```

Comment:
Why do we need this `regex` library? Is it better than Python's builtin `re` package?

Reply:
I actually removed what was using regex, so neither are needed anymore. 
But to answer your question, I forgot that Python had it a built in

Reply:
Python is "batteries included" for many things.. :)